build.xml

```diff
@@ -6,6 +6,8 @@
   <property file="build.properties"/>
   <property name="compile.java.version" value="1.6"/>
   <property name="compile.java.bootclasspath" value="${build.bootclasspath}"/>
+  <!-- A class name from the release after the desired version given above. -->
```

Comment:
/the release/a release/

Maybe add "This class will NOT be found".

Reply:
It really should be the next release. I was trying not to waste another line of comment on such a minor thing, but I'll expand on this.

Reply:
Yes, please.  

This property name is only used once.  Maybe it should be next to its usage.  One thing that would be alleviated by putting closer to usage is the name "newerClassname" does not convey intent sufficiently all alone ontop of file.  Otherwise a fuller comment, or a really descriptive, and possibly very lengthy name.

Reply:
I explicitly do not want it where it's used, because the usage is boilerplate and does not need to adjusted, and the correct value for this property depends on the value of compile.java.version, set two lines above.

Reply:
OK. 

How about s/newerClassname/classShouldNotExist/ ?

How about much richer comment spelling out this lore:

This build file targets specific JVM version. The below class does not exist in targeted JVM version. It does exist in later versions. We use this class to check provided bootclasspath jars are of correct version.

Reply:
I had tried changing it to unavailableClassname, but I think that makes the actual code in -real-build look funny. I stuck with newerClassname and an expanded explanation.